OtripleS.Web.Api/Controllers/StudentController.cs

```diff
@@ -25,5 +27,16 @@ public async ValueTask<ActionResult<Student>> DeleteStudentAsync(Guid studentId)
 
             return Ok(storageStudent);
         }
+
+        [HttpPut("{studentId}")]
+        public async ValueTask<ActionResult<Student>> UpdateStudentAsync(Guid studentId,
+            [FromBody] StudentUpdateDto dto)
+        {
+            var student = await this.studentService.ModifyStudentAsync(studentId, dto);
+
+            if (student.HasValue()) return Ok(student);
+
+            return BadRequest("update student failed");
```

Comment:
controllers should not contain any logic/flow control 
Also exceptions here are not fully covered, review this for examples:
https://github.com/hassanhabib/SchoolEM/blob/master/SchoolEM/Controllers/StudentsController.cs

Reply:
fixed